src/JLSOFile.jl

```diff
@@ -45,8 +46,15 @@ function JLSOFile(
     end
 
     _versioncheck(version, WRITEABLE_VERSIONS)
-    objects = Dict{String, Vector{UInt8}}()
-    jlso = JLSOFile(version, julia, format, compression, image, _pkgs(), objects)
+    jlso = JLSOFile(
+        version,
+        julia,
+        format,
+        compression,
+        image,
+        Pkg.TOML.parse.(_env())...,
+        Dict{Symbol, Vector{UInt8}}()
```

Comment:
I thjink it would be get much better compression if it was a single compressed stream not a vector of them as that is where a lot of the resundancy is.
Common substrings.

Reply:
I'm not sure I follow what you mean... can you provide an example? We need to store the bytes for each object stream independently to support lazy loading of objects.

Reply:
Oh I thought this was Manifest.
This is fine